docs/python-sdk/dynamic_log_levels.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Dynamic Log Levels
+sidebar_label: Dynamic Log Levels
+---
+
+## Setting Dynamic Log Levels
+
+A simple example here is worth a million words.
+
+First add a simple module.
```

Comment:
You didn't introduce it, but this "simple" repetitiveness isn't great. Can you drop the second instance of "simple" here and in the ruby and elixir docs?